### ISSUE TITLE
[BUG] Zip file for voice recording download is broken

### ISSUE THREAD

**Issue by laubonghaudoi** [common-voice/common-voice#4253]
We received multiple reports from our volunteers that the zip file for downloading recordings is corrupted or broken. As you can see from the screenshot below, the volunteer contributed 4013 recordings, yet when she requested to download her recordings, the zip file generated has only 0.39 MB in size. When downloaded and unzipped, there are only 13 recordings inside.
![telegram-cloud-photo-size-1-5127565797915470985-y](https://github.com/common-voice/common-voice/assets/11172180/32052e6c-8c55-41c4-98b2-2e04fc4a30e7)

So among 4k+ recordings the zip only includes 13 of them, which is definitely not WAI. Please fix this, or our volunteers are unable to contribute more. I am not sure if this is related to https://github.com/common-voice/common-voice/issues/4241#issuecomment-1822333646 Please let me know what I can do on our end. Thank you.

**Comment by moz-dfeller** [comment]
Hi @laubonghaudoi ! Thank you for flagging this. I updated the code that downloads the clips here #4321. Is it possible to find out whether the people with broken or corrupted clips are still experiencing this issue? Other community members were able to download > 2k clips without any problems now.

**Comment by laubonghaudoi** [comment]
Thank you so much @moz-dfeller. I am asking the contributors to try downloading again. Will post update here whether this is fixed or not.

**Comment by moz-dfeller** [comment]
You're welcome @laubonghaudoi !

> I believe the contributors were downloading ~4000 clips, which was too large and had to be split into two zip files.

Yeah we split at 2k clips into multiple zip files. That's why I was asking members with more than 2k clips to see if files get corrupted when clips have to be split.

Keep me posted! 🙂

**Comment by laubonghaudoi** [comment]
Hi @moz-dfeller , so the contributors got back to me and said that now they are stuck at  https://github.com/common-voice/common-voice/issues/4241 . They hit the refresh button and nothing happened. And some of their last download request was in November last year. They can't make a new request because this last one is still being processed. So I'd say we still can't verify the download yet, and this is blocked by https://github.com/common-voice/common-voice/issues/4241

**Comment by moz-dfeller** [comment]
Thanks for asking @laubonghaudoi ! I will check which processes are stuck and delete them. Let me get back to you once I cleared the stuck download requests.

**Comment by laubonghaudoi** [comment]
Hi are the download requests cleared? Our volunteers reported that they are still having this issue.

**Comment by jessicarose** [comment]
Thanks so much for your patience on this, investigations haven't found any stuck processes in the database to delete, which we hoped would resolve this issue. Could I ask you to having someone impacted email us at commonvoice@mozilla.org with "stuck downloads" in the subject line, as knowing the email address associated with a stuck account (or all impacted accounts, if that were possible) would let us further narrow our investigations into this issue.

**Comment by laubonghaudoi** [comment]
Hi @jessicarose , sorry for the late response, I am still collecting the full list of contributors who are stuck on this. One example I collected is zhoujiaxin668@outlook.com, account name `周嘉欣`. This screenshot shows that she is still stuck at her last download request 2023-11-08 15:57. Can you take a look at this account?
<img width="960" alt="707235040443303666" src="https://github.com/common-voice/common-voice/assets/11172180/9f8f0f58-1809-4e85-8247-eeb57751bcfe">

**Comment by jessicarose** [comment]
Hello hello, I think that this has just been located and fixed for effected users. Could I ask you to redact the email address for this user and let me know if they're now able to request a new download link? Let me know how this works and if this is still blocked we'll get right back to work on it!

**Comment by moz-kathyreid** [comment]
Hi there @laubonghaudoi - I am reviewing open Issues to see which ones are stale and which ones are still occurring. Can you let me know if this issue with downloading zip files is still occurring? Thank you in advance.

**Comment by moz-bozden** [comment]
I'm closing this as stale. At that time we tested the fix and it was working.
Please open a new issue if something is still off...
